src/Chapter09.Tests/Listing09.04.DeclaringRecordClass.Tests.cs

```diff
@@ -0,0 +1,50 @@
+namespace AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Listing09_04.Tests;
+using AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Listing09_01;
+using AddisonWesley.Michaelis.EssentialCSharp.Chapter09.Tests;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+[TestClass]
+public partial class CoordinateTests
+{
+    [TestMethod]
+    public void Create_Coordinate_IsReadOnly()
+    {
+        Coordinate coordinate = new(
+            new Angle(180, 0, 0), new Angle(180, 0, 0));
+    }
```

Comment:
This test content does not actually test that a coordinate is readonly. That seems like something we would test with a CompilerAssert. Is that what is intended here?

Reply:
This needs to be a CompilerAssert test.

Reply:
I have deleted it for the time being since CompilerAssert doesn't currently support testing a code snippet in the test.